README.md

```diff
@@ -5,10 +5,27 @@
 
 The Yellow Paper is a formal definition of the Ethereum protocol, originally by Gavin Wood, currently maintained by Nick Savers and with contributions from many people around the world.
 
-It is free culture work, licensed under the Creative Commons Attribution Share-Alike (CC-BY-SA)i version 4.0.
+It is free culture work, licensed under the Creative Commons Attribution Share-Alike (CC-BY-SA) version 4.0.
 
-## Usage
+## Releases
+This paper currently does not include changes to Ethereum after Byzantium, however if you like, feel free to make a PR! For further information on previous and planned releases, see [here](https://github.com/ethereum/wiki/wiki/Releases).
 
-The paper comes as a single ``latex`` file ``Paper.tex``. The latest version is generally available as a PDF at https://ethereum.github.io/yellowpaper/paper.pdf or just [yellowpaper.io](http://yellowpaper.io/) .
+## The paper
 
-In order to build, use the supplied build script (``build.sh``). Following the first time, you can use the standard `pdflatex` tools like http://latex.informatik.uni-halle.de/latex-online/latex.php for compiling/preview. It can be viewed in ``PDF`` format after building.
+[Here](https://github.com/ethereum/yellowpaper/blob/master/Paper.pdf) is the latest version of the PDF (which is the same as the Paper.pdf file above). This version of the yellow paper can be used to get links that direct to specific info in the yellow paper, which is useful e.g. for pedagogical and academic purposes. [Here](
+https://github.com/ethereum/yellowpaper/files/1596574/Paper.pdf#blockhash) is an example of such a use. To get this link you just right click on a link that directs to the target that you want in the PDF (in a browser or program) and copy the link.
```

Comment:
There is no https://github.com/ethereum/yellowpaper/files/*

Reply:
Does this link not work for you? It works for me.

Reply:
https://github.com/ethereum/yellowpaper/files/1596574/Paper.pdf#blockhash

Reply:
I guess a problem is that if you clone the repo, and build, then you'll have two PDFs. So you'd have to delete the original PDF before building and pushing the modified changes. But I don't see this as a big deal, and I still think it is a net benefit to include the PDF in the repo.

Reply:
That link works... what is this wizardry?
Oh by the way, this one shouldn't work as discussed earlier.
https://github.com/ethereum/yellowpaper/blob/master/Paper.pdf

Reply:
Haha, it's the hyperref package. You create a hypertarget with \hypertarget{foo} (best to put it in the line before what you want a link to direct to). Then you create a link with \hyperref{foo}. Then you get the link by right clicking on the link and copying the link! OK I'll remove that link.